README.md

```diff
@@ -86,14 +86,18 @@ It's best to have at least 30GB of free space for Docker containers and images.
 
 ## Releasing a new version
 
-If you want to release a new version of YamUI, do the following to create a release branch:-
+If you want to release a new version of YamUI, do the following to create a release branch. Replace version_type with `major`, `minor`
+or `patch` as appropriate, based on [SemVer](https://semver.org):-
+
 
 ```sh
 git checkout -b awesome-release-branch
-npm version patch -m "Release version %s"
+npm version version_type -m "Release version %s"
 git push
```

Comment:
should probably also add `git push --tags` after `git push`

Reply:
Ah, interesting. Is that needed instead or in addition to the git push?

Reply:
From reading git manual stuffs, seems its needed as a separate step. Updated.